tests/README.md

```diff
@@ -91,6 +91,15 @@ Notes:
   - Authorization V2 scenarios only support PowerFlex
   - Upgrade from Authorization V1 to V2 is not supported. Only V1 to other V1 versions is allowed.
 
+### HBNFS Prerequisites
+
+If running the HBNFS e2e tests, further setup must be done:
+
+- On each of the worker nodes, setup the nfs-server service and ensure that mounting with nfs4 is enabled.
+- Install the `sshpass` command on the VM which the tests are being run.
```

Comment:
Minor suggestion: Install sshpass command on the VM where the tests are being run.